dotcom-rendering/src/components/marketing/epics/ctas/ContributionsEpicButtons.tsx

```diff
@@ -189,18 +190,27 @@ export const ContributionsEpicButtons = ({
 	}
 
 	const getChoiceCardCta = (cta: Cta): Cta => {
-		if (
-			showChoiceCards &&
-			variantOfChoiceCard === 'US_THREE_TIER_CHOICE_CARDS'
-		) {
+		/** In the US - direct 50 % traffic to the checkout page and 50 % traffic to the landing page for the AB test  */
+
+		if (showChoiceCards && countryCode === 'US') {
+			const oneOffUrl =
```

Comment:
`oneOffUrl` can be removed if you just pass the relevant urls into `addChoiceCardsParams`/`addChoiceCardsOneTimeParams` below

Reply:
Removed the oneOffUrl